.github/workflows/php.yml

```diff
@@ -19,9 +19,20 @@ jobs:
   php-cs-fixer:
     name: PHP-CS-Fixer
     runs-on: ubuntu-latest
+    strategy:
+        matrix:
+            php: ['7.3', '7.4']
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2.0.0
+        -   name: Setup PHP
+            uses: shivammathur/setup-php@v2
+            with:
+                php-version: ${{ matrix.php }}
+
+        -   name: Checkout
+            uses: actions/checkout@v2
+
+        -   name: Composer Install
+            run: composer install --ansi --prefer-dist --no-interaction --no-progress --quiet
 
-      - name: Run PHP-CS-Fixer
-        uses: prestashopcorp/github-action-php-cs-fixer@master
+        -   name: Run PHPCSFixer
```

Comment:
Any reason to remove the spaces in PHP CS Fixer ?

Reply:
No, I can restore them ^^